src/app/admin/usersTable/UsersTable.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+
+import React, { useState } from "react";
+import Table, { ColumnDisplayFunction, Row, TableHeaders } from "@/components/Tables/Table";
+import styled from "styled-components";
+import Modal from "@/components/Modal/Modal";
+import { deleteUser, UserRow } from "@/app/admin/adminActions";
+import Button from "@mui/material/Button";
+import DeleteIcon from "@mui/icons-material/Delete";
+import RefreshPageButton from "@/app/admin/RefreshPageButton";
+
+const DangerDialog = styled(Modal)`
+    & #deleteUserDialog {
+        background-color: ${(props) => props.theme.error};
+    }
+`;
+
+export const OptionButtonDiv = styled.div`
+    display: flex;
+    padding-top: 1rem;
+    gap: 1rem;
+    justify-content: center;
+`;
+
+const usersTableHeaderKeysAndLabels: TableHeaders = [
+    ["id", "USER ID"],
```

Comment:
please 'normal case' the headers and set capitalisation in CSS (screen readers do much better with this!)

Reply:
I've just decapitalised these for just now